### PR TITLE
test: integration test for add and drop index

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
integration test for add and drop index

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: tests/multi_source/main.go [hunk 1]
```diff
@@ -165,14 +167,35 @@ func truncateDDL(ctx context.Context, db *sql.DB) {
...
 		if err == nil {
-			success++
-			if success%100 == 0 {
-				log.S().Info(id, " success: ", success)
+			insertSuccess++
+			if insertSuccess%100 == 0 {
+				log.S().Info(id, " insert success: ", insertSuccess)
+			}
+		}
+		if err != nil && !strings.HasPrefix(err.Error(), "Error 1146:") {
```
Comment (amyangfei) [inline tests/multi_source/main.go:182]: In which case will `Error 1146:` happen?

Comment (leoppro) [reply tests/multi_source/main.go:182]: the `Error 1146:` will happen when the schema or table is dropped in case `createDropSchemaDDL`.

**Comment by leoppro** [comment]
@liuzix  PTAL

**Comment by lance6716** [comment]
LGTM

**Pull request merged**